development/compiling/compiling_with_mono.rst

```diff
@@ -68,8 +68,9 @@ This path must be ``modules/mono/glue`` in the Godot directory::
 
     <godot_binary> --generate-mono-glue modules/mono/glue
 
-This command will tell Godot to generate the file ``modules/mono/glue/mono_glue.gen.cpp``
-and the C# solution for the Godot API at ``modules/mono/glue/Managed/Generated``.
+This command will tell Godot to generate the file ``modules/mono/glue/mono_glue.gen.cpp``,
+the C# solution for the Godot API at ``modules/mono/glue/GodotSharp/GodotSharp/Generated``,
+and the C# solution for the editor tools at ``modules/mono/glue/GodotSharp/GodotSharpEditor/Generated``.
```

Comment:
GodotSharpEditor is still Godot API though, but editor-only (not available in exported games), like `EditorInspector`. The wording `editor tools` makes me think of `modules/mono/editor/GodotTools` which contains the C# solution for the Godot plugin, etc.

Reply:
Correct. We may want to change to better naming before 4.0.